lib/matplotlib/tests/test_text.py

```diff
@@ -1204,6 +1204,19 @@ def test_ytick_rotation_mode():
     plt.subplots_adjust(left=0.4, right=0.6, top=.99, bottom=.01)
 
 
+@image_comparison(baseline_images=['features.png'], remove_text=False, style='mpl20')
+def test_text_features():
+    fig = plt.figure(figsize=(5, 1.5))
+    t = fig.text(0, 0.7, 'Default: fi ffi fl st', fontsize=32)
```

Comment:
Can you pad out the label strings to the same size so that the first letters of the example text line up?  Does not matter for the machine to check, but makes it a bit easier for the humans to check the differences.

Reply:
Do you want it aligned in the input text, or in the resulting image? If I align it in the input, then it's not quite right in the result because it's not monospace:
<img width="500" height="150" alt="features" src="https://github.com/user-attachments/assets/18f4d481-5614-4f50-8df1-ca53a318e0c8" />
but I could change to right alignment, which would be close, I think?

Reply:
I spend so much time looking at monospace fonts I forget not all fonts are monospace 🤦🏻 . 

Right-align is exactly what I wanted.